src/SWP/TemplateEngineBundle/Service/ContainerService.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Template Engine Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\TemplateEngineBundle\Service;
+
+use SWP\TemplateEngineBundle\Container\SimpleContainer;
+use SWP\TemplateEngineBundle\Model\Container;
+use SWP\TemplateEngineBundle\Model\ContainerData;
+
+class ContainerService
+{
+    const OPEN_TAG_TEMPLATE = '<div id="swp_container_{{ id }}" class="swp_container {{ class }}" style="{% if height %}height: {{ height }}px;{% endif %}{% if width %}width: {{width}}px;{% endif %}{{styles}}"{% for value in data %} data-{{value.getKey()}}="{{value.getValue()}}"{% endfor %} >';
```

Comment:
I would put it into some "template" file and not defining it as constant in the code. you will be able to change it if needed without modifying the code.